etc/gatein/portal/portal/picketlink-idm-LDAP-config.xml.template

```diff
@@ -0,0 +1,378 @@
+            <name>organization_type</name>
+            <relationships>
+              <relationship>
+                <relationship-type-ref>JBOSS_IDENTITY_MEMBERSHIP</relationship-type-ref>
+                <identity-object-type-ref>USER</identity-object-type-ref>
+              </relationship>
+              <relationship>
+                <relationship-type-ref>JBOSS_IDENTITY_MEMBERSHIP</relationship-type-ref>
+                <identity-object-type-ref>organization_type</identity-object-type-ref>
+              </relationship>
+            </relationships>
+            <credentials/>
+            <attributes/>
+            <options>
+              <option>
+                <name>idAttributeName</name>
+                <value>cn</value>
+              </option>
+              <option>
+                <name>ctxDNs</name>
+                <value>ou=Organization,o=portal,o=gatein,${USER_DIRECTORY_BASE_DN}</value>
+              </option>
+              <!--<option>-->
+              <!--<name>entrySearchFilter</name>-->
+              <!--<value></value>-->
+              <!--</option>-->
+              <option>
+                <name>allowCreateEntry</name>
+                <value>true</value>
+              </option>
+              <option>
+                <name>parentMembershipAttributeName</name>
+                <value>member</value>
+              </option>
+              <option>
+                <name>isParentMembershipAttributeDN</name>
+                <value>true</value>
+              </option>
+              <option>
+                <name>allowEmptyMemberships</name>
+                <value>true</value>
+              </option>
+              <option>
+                <name>createEntryAttributeValues</name>
+                <value>objectClass=top</value>
+                <value>objectClass=groupOfNames</value>
+              </option>
+            </options>
+          </identity-object-type>
+        </supported-identity-object-types>
+        <options>
+          <option>
+            <name>providerURL</name>
+            <value>ldap://localhost:${DEPLOYMENT_LDAP_PORT}</value>
+          </option>
+          <option>
+            <name>adminDN</name>
+            <value>${USER_DIRECTORY_ADMIN_DN}</value>
+          </option>
+          <option>
+            <name>adminPassword</name>
+            <value>${USER_DIRECTORY_ADMIN_PASSWORD}</value>
+          </option>
+          <option>
+            <name>searchTimeLimit</name>
+            <value>10000</value>
+          </option>
+          <option>
+            <name>createMissingContexts</name>
+            <value>true</value>
+          </option>
+          <option>
+            <name>customJNDIConnectionParameters</name>
+            <value>com.sun.jndi.ldap.connect.pool=true</value>
+          </option>
+          <option>
+            <name>customSystemProperties</name>
+            <value>com.sun.jndi.ldap.connect.pool.maxsize=300000</value>
```

Comment:
This value should not be kept as default, it's way too much, there is a confusion between timeout and pool size cf https://jira.exoplatform.org/browse/EXOGTN-2398